metrics/ci_integrations/docs/diagrams/ci_integrations_config_validator_class_diagram.puml

```diff
@@ -95,13 +106,16 @@
 
     CoolIntegrationDestinationValidationDelegate ..|> DestinationValidationDelegate
 
-    CoolIntegrationDestinationConfigValidator --|> ConfigValidator
+    CoolIntegrationDestinationConfigValidator -left-|> ConfigValidator
     CoolIntegrationDestinationConfigValidator -left-> CoolIntegrationDestinationValidationDelegate : uses
-    CoolIntegrationDestinationValidationDelegate -down-> CoolIntegrationClient : uses
+    CoolIntegrationDestinationValidationDelegate -left-> CoolIntegrationClient : uses
     CoolIntegrationConfigValidatorFactory ..|> ConfigValidatorFactory
     CoolIntegrationConfigValidatorFactory --> CoolIntegrationDestinationConfigValidator : creates
-    CoolIntegrationConfigValidatorFactory --> CoolIntegrationClient : creates
+    CoolIntegrationConfigValidatorFactory -left-> CoolIntegrationClient : creates
     CoolIntegrationConfigValidatorFactory --> CoolIntegrationDestinationValidationDelegate : creates
     CoolIntegrationDestinationConfigValidator --> ConfigValidationError : throws
 
+    ValidateCommand -left-> IntegrationParty : uses
+    ValidateCommand -left-> ConfigValidatorFactory : uses
```

Comment:
Does the `validate` command uses the `ConfigValidator` abstract class to validate the config? Also, it seems like this class uses the `Config` interface.